workflow/tools/util_in_docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM java
```

Comment:
this base image seems to be old, see this: https://hub.docker.com/_/java?tab=reviews